inspec.yml

```diff
@@ -1,5 +1,8 @@
-name: vsphere
-title: VMware vSphere/ESXi Resource Pack
-maintainer: chris@lollyrock.com
-summary: This resource pack provides compliance resources for VMware vSphere/ESXi
+name: inspec-powercli
+title: InSpec PowerCLI
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: Apache-2.0
+summary: An InSpec Compliance Profile
 version: 0.1.0
```

Comment:
This needs to bump to 1.x since it breaks with the previous resource pack. We also should tag the latest release and bump the version to 0.1.1 before we apply this PR.

Reply:
👍

Reply:
The name changed too I believe. Is that alright?